New serverless pattern - dynamo-pipes-sqs-cdk

twonder opened the pull request:
*Issue #1020*

This pattern demonstrates a low code way to send DynamoDB stream records to an SQS queue using EventBridge Pipes.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

singledigit reviewed (changes requested):
Tim,

The `example-pattern.json` file is not validating. I believe you are missing the `templateFile` value. Please correct and update.

twonder commented:
Thanks Eric, I updated the example-pattern.json to include a templateFile which is the C# class that has the stack definition.  Hopefully that works even though it isn't a true template file.

I also tried to run the scripts/validate.js script locally and it seemed like the language C# is supposed to be .NET so I updated that as well.

Are there instructions on running the scripts/validate.js file, or is that not something contributors are supposed to run?  If not, it seems like it would be helpful to put in the README to validate before creating the PR.
